instrumentation/netty/netty-4.0/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/netty/v4_0/DataCaptureUtils.java

```diff
@@ -24,9 +24,12 @@
 import io.netty.util.Attribute;
 import io.netty.util.AttributeKey;
 import io.opentelemetry.api.trace.Span;
-import java.io.ByteArrayOutputStream;
-import java.io.UnsupportedEncodingException;
+import java.io.*;
```

Comment:
We should not use * in imports in javaagent.

Reply:
Oh yes, correct. Will correct